autoload/gradle.vim

```diff
@@ -283,33 +283,36 @@ endfunction
 " the current Android project.
 function! gradle#setClassPath()
 
-  let l:jarList = []
-  let l:srcList = []
-
-  let l:oldJars = split($CLASSPATH, gradle#classPathSep())
-  let l:oldSrcs = split($SRCPATH, ",")
+  if exists(":JCstart")
+    return
+  else
```

Comment:
Else after a return is not necessary and adds useless indentation. Just endif it here and keep the rest of the function without modification.